### PR TITLE
Update "super" usage

### PR DESCRIPTION
**Pull request by rahulporuri**
This PR updates `super` usage using automated regex-based search and replace. After the changes were made, we reviewed each of the changes to ensure that we were calling the right class before.

See https://github.com/enthought/traits/pull/1280

### CODE REVIEW

File: enable/container.py [hunk 1]
```diff
@@ -574,12 +574,12 @@ def _fit_window_changed(self, old, new):
     def _bounds_changed(self, old, new):
         # crappy... calling our parent's handler seems like a common traits
         # event handling problem
```
Comment (aaronayres35) [inline enable/container.py:576]: I was thinking this same thing when reviewing recent chaco PRs...

File: enable/scrolled.py [hunk 2]
```diff
@@ -612,7 +612,7 @@ def _container_handle_mouse_event(self, event, suffix):
     # -------------------------------------------------------------------------
 
     def __getstate__(self):
-        state = super(Scrolled, self).__getstate__()
+        state = super().__getstate__()
         for key in ["alternate_vsb", "_vsb", "_hsb"]:
             if key in state:
                 del state[key]
```
Comment (aaronayres35) [inline enable/scrolled.py:618]: orthogonal but these could be updated to use `transient` metadata

**Review by aaronayres35 — APPROVED** [review]
LGTM

File: enable/scrolled.py (hunk 2 above)
Comment (rahulporuri) [reply enable/scrolled.py:618]: i'll open a separate issue/PR for this.

Comment (rahulporuri) [reply enable/scrolled.py:618]: i created https://github.com/enthought/enable/issues/795

**Pull request merged**